packages/fether-electron/src/main/app/methods/setupAppListeners.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import { productName } from '../../../../electron-builder.json';
+import { getSavedWindowPosition } from '../settings';
+import Pino from '../utils/pino';
+
+const pino = Pino();
+
+function setupAppListeners () {
+  this.fetherApp.on('create-app', () => {
```

Comment:
Prefer to pass fetherApp as argument to the function. Only use `this` inside a class